test/e2e/specs/scaleupdown.go

```diff
@@ -0,0 +1,173 @@
+package specs
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	apiappsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	internalapi "github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/util/managedcluster"
+	"github.com/openshift/openshift-azure/pkg/util/randomstring"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/clients/openshift"
+)
+
+var _ = Describe("Scale Up/Down E2E tests [ScaleUpDown][Real][LongRunning][EveryPR]", func() {
+	const (
+		sampleDeployment = "hello-openshift"
+	)
+	var (
+		azurecli          *azure.Client
+		occli             *openshift.Client
+		scaleUpManifest   = flag.String("scaleUpManifest", "../../test/manifests/normal/scaleup.yaml", "Path to the scale up manifest to send in a partial update request to the RP")
```

Comment:
There is probably no point in using `flag` now.

Reply:
however I think it doesn't hurt to leave this here. MSFT might want to use different partial update files (for example)